src/Bridge/Symfony/Bundle/Command/SwaggerCommand.php

```diff
@@ -78,10 +80,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         /** @var string $version */
-        $version = $input->getOption('spec-version');
+        $version = $input->getOption('spec-version')[0];
```

Comment:
It will fail if you pass an empty string.

```suggestion
        $version = explode('.', $input->getOption('spec-version'));
```